### ISSUE TITLE
webpack-chunks.json configurable request

### ISSUE THREAD

**Issue by lonelyclick** [halt-hammerzeit/universal-webpack#42]
I have two entrys in my webpack file and start it separately, but webpack-chunks.json always be covered，so at 'universal.settings.js'：

```
module.exports = {
  server: {input: 'xxx', output: 'xxx'}
  chunkFilename: 'xxxx-webpack-chunk.json' 
}
```

please support it #43

**Comment by halt-hammerzeit** [comment]
Explain more about your situation: why do you have two entries in Webpack configuration.
I can see why some people may have 2 entries: for code splitting (e.g. "photos", "blog", "forum").
But what's your case?

**Comment by lonelyclick** [comment]
I have two react + Redux applications, A and B, A is an independent application with a AppHeader component. AppHeader contains a number of complex components, these components need API organization Redux data, you want to split the AppHeader is more difficult. B is a different application, it needs AppHeader and most of the logic is not consistent with A. So I can only put A and B into a repo, make two entrances. And later there will be more similar to the B application, even with the splitting code separately, it will become increasingly difficult to maintain.

**Comment by halt-hammerzeit** [comment]
Published `universal-webpack@0.1.41`
